pkg/virtctl/imageupload/imageupload.go

```diff
@@ -132,8 +133,9 @@ func NewImageUploadCommand(clientConfig clientcmd.ClientConfig) *cobra.Command {
 	cmd.Flags().StringVar(&pvcSize, "pvc-size", "", "DEPRECATED - The size of the PVC to create (ex. 10Gi, 500Mi).")
 	cmd.Flags().StringVar(&size, "size", "", "The size of the DataVolume to create (ex. 10Gi, 500Mi).")
 	cmd.Flags().StringVar(&storageClass, "storage-class", "", "The storage class for the PVC.")
-	cmd.Flags().StringVar(&accessMode, "access-mode", "ReadWriteOnce", "The access mode for the PVC.")
-	cmd.Flags().BoolVar(&blockVolume, "block-volume", false, "Create a PVC with VolumeMode=Block (default Filesystem).")
+	cmd.Flags().StringVar(&accessMode, "access-mode", "ReadWriteOnce", "The access mode for the PVC, either ReadWriteOnce or ReadWriteMany.")
```

Comment:
It's good to improve the command line help, but it is not relevant to the new command line you are adding. It made it harder for me to review this commit.

Reply:
Thanks, Dan. I understood your point now, removing it from here.

Reply:
I don't think you have. You still modify the access mode line.

Reply:
The access mode line is in my forgotten zone, will revert it.